projects/scion/microfrontend-platform/src/lib/ɵmessaging.model.ts

```diff
@@ -92,6 +92,18 @@ export enum PlatformTopics {
   Applications = 'ɵAPPLICATIONS',
 }
 
+/**
+ * Declares internal platform topics.
+ */
+export namespace PlatformTopics {
```

Comment:
What do you mean by that specifically? By defining constants instead of enum values? Actually, we are merging the functions into the `enum` namespace (declaration merging). But that might be a little confusing. Is that your point?

Reply:
Yes I would rather just have the `namespace PlatformTopics` than both an `enum` and a `namespace`.

Reply:
Done, moved the `enum` values as `constants` to the namespace, as follows:
```ts
/**
 * Declares internal platform topics.
 */
export namespace PlatformTopics {
  /**
   * Allows requesting the subscription count on a topic.
   */
  export const RequestSubscriberCount = 'ɵREQUEST_SUBSCRIBER_COUNT';
  /**
   * When a client gains the focus it publishes a retained event to this topic.
   */
  export const FocusIn = 'ɵFOCUS_IN';
  /**
   * Allows testing whether the requester has received focus or contains embedded web content that has received focus.
   */
  export const IsFocusWithin = 'ɵIS_FOCUS_WITHIN';
  /**
   * Allows reading the platform properties from this retained topic.
   */
  export const PlatformProperties = 'ɵPLATFORM_PROPERTIES';
  /**
   * Allows reading the registered applications from this retained topic.
   */
  export const Applications = 'ɵAPPLICATIONS';
  /**
   * Computes the topic where a client can publish its heartbeat.
   */
  export function heartbeat(clientId: string): string {
    return `ɵclient/${clientId}/heartbeat`;
  }
}

```